Update automatic-vm-guest-patching.md - PowerShell command error

mbsnl opened the pull request:
Update automatic-vm-guest-patching.md - PowerShell command for enabling/updating VM is wrong, using "Get-AzVM" you get the VM Object in the $VirtualMachine variable, after that you can perform any PowerShell commands with the "-VM $VirtualMachine" parameter, like the Set-AzVM* or the Update-AzVM*. Further the "Get-AzVM" command has no parameters for Credentials, OperatingSystem, ComputerName (just Name) or "VM". Also small update to the command "" for VM Status to have consistancy on this page to avoid user mistakes.

mbsnl committed:
Update automatic-vm-guest-patching.md - PowerShell command error

Update automatic-vm-guest-patching.md - PowerShell command for enabling/updating VM is wrong, using "Get-AzVM" you get the VM Object in the $VirtualMachine variable, after that you can perform any PowerShell commands with the "-VM $VirtualMachine" parameter, like the Set-AzVM* or the Update-AzVM*. Further the "Get-AzVM" command has no parameters for Credentials, OperatingSystem, ComputerName (just Name) or "VM".
Also small update to the command "" for VM Status to have consistancy on this page to avoid user mistakes.

mirichmo commented on articles/virtual-machines/automatic-vm-guest-patching.md:
```diff
@@ -295,7 +297,9 @@ GET on `/subscriptions/subscription_id/resourceGroups/myResourceGroup/providers/
-Get-AzVM -ResourceGroupName "myResourceGroup" -Name "myVM" -Status
+$VirtualMachineName='myVM'
+$ResourceGroupName='myResourceGroup'
+Get-AzVM -ResourceGroupName $ResourceGroupName -Name $VirtualMachineName -Status -OutVariable VirtualMachine
```
This change is unnecessary. Please leave it as-is.

mirichmo commented on articles/virtual-machines/automatic-vm-guest-patching.md:
```diff
@@ -239,7 +239,9 @@ Set-AzVMOperatingSystem -VM $VirtualMachine -Windows -ComputerName $ComputerName
-Get-AzVM -VM $VirtualMachine -Windows -ComputerName $ComputerName -Credential $Credential
+$VirtualMachineName='myVM'
+$ResourceGroupName='myResourceGroup'
+$VirtualMachine=Get-AzVM -ResourceGroupName $ResourceGroupName -Name $VirtualMachineName
```
Thanks for noticing the syntax error and submitting a PR. I'd like to see it updated to conform to syntax of the other Get's within the documentation:

$VirtualMachine = Get-AzVM -ResourceGroupName "myResourceGroup" -Name "myVM"

mbsnl committed:
Update automatic-vm-guest-patching.md - PowerShell command error

Update automatic-vm-guest-patching.md - PowerShell command for enabling/updating VM is wrong, using "Get-AzVM" you get the VM Object in the $VirtualMachine variable, after that you can perform any PowerShell commands with the "-VM $VirtualMachine" parameter, like the Set-AzVM* or the Update-AzVM*.
